IPython/extensions/autoreload.py

```diff
@@ -552,21 +605,45 @@ def autoreload(self, parameter_s=""):
           autoreloaded.
 
         """
-        if parameter_s == "":
+        args = magic_arguments.parse_argstring(self.autoreload, line)
+        mode = args.mode.lower()
+
+        def p(msg):
+            print(msg)
```

Comment:
Worth making it one func def and one call on each reload less?

```suggestion
        p  = print
```